conda_build/build.py

```diff
@@ -1194,9 +1198,9 @@ def create_info_files(m, files, prefix):
     write_info_files_file(m, files)
 
     files_with_prefix = get_files_with_prefix(m, files, prefix)
+    files_with_prefix = record_prefix_files(m, files_with_prefix)
```

Comment:
This broke `prefix_placeholder` recording in `paths.json`.
I don't know what the proper fix is and can't investigate this currently.
xref: https://github.com/conda-forge/conda-build-feedstock/pull/131

Reply:
Also, when someone fixes this, I strongly suggest to add a test that checks for the `prefix_placeholder` entries to avoid regressions.